sheepdog/transactions/upload/entity.py

```diff
@@ -491,7 +492,7 @@ def _verify_node_project_id(self, node):
         return node.project_id == self.transaction.project_id
 
     def get_metadata(self):
-        metadata = {'project_id': self.transaction.project_id}
+        metadata = {'dbgap_accession_numbers': flask.g.dbgap_accession_numbers}
```

Comment:
can you rename the key to just `acl` ? fence shouldn't know that it's `dbgap_access_numbers`